iotlabcli/tests/helpers_test.py

```diff
@@ -102,53 +104,71 @@ def fake_cmd():
                         in str(warn[-1].message))
 
 
-class TestFilesDict(unittest.TestCase):
-    """Test FilesDict class."""
+# Test FilesDict class
 
-    def test_no_overwrite(self):
-        """Test FilesDict values overriding."""
-        file_dict = helpers.FilesDict()
-        file_dict['a'] = 1
-        # Can re-add same value
-        file_dict['a'] = 1
+@pytest.fixture(scope='function')
+def file_dict():
```

Comment:
Why using a fixture here, it is not as if it was as a prepared mocked environment, the tests are actually testing the class so make sense to initialize it in the test.

Reply:
it's either a fixture and file_dict everywhere or repeating helpers.FilesDict() everywhere, either way there's repetition. Just going pytest all the way I guess.